resources/views/accounts/_edit_country.blade.php

```diff
@@ -0,0 +1,43 @@
+{{--
+    Copyright (c) ppy Pty Ltd <contact@ppy.sh>. Licensed under the GNU Affero General Public License v3.0.
+    See the LICENCE file in the repository root for full licence text.
+--}}
+@php
+    use App\Libraries\User\CountryChangeTarget;
+    use App\Models\Country;
+
+    $countryChangeTarget = CountryChangeTarget::get($user);
+@endphp
+<div class="account-edit-entry account-edit-entry--read-only">
+    <div class="account-edit-entry__label account-edit-entry__label--top-pinned">
+        {{ osu_trans('accounts.edit.profile.country') }}
+    </div>
+    <div>
```

Comment:
could do with some better wrapping and flag centering 
![image](https://github.com/ppy/osu-web/assets/1694544/018908a8-e74b-4601-a960-70875ddf2632)